onnxscript/converter.py

```diff
@@ -753,6 +756,7 @@ def translate_call_expr(self, node):
         args = [self.translate_opt_expr(x) for x in node.args]
         args = autocast.static_cast_inputs(self, callee.get_schema(), *args)
         attrs = [self.translate_attr(x.arg, x.value) for x in node.keywords]
+        attrs = [attr for attr in attrs if attr is not None]
```

Comment:
It'd be nice to explain the filter with a comment

Reply:
Done